demo/src/app.js

```diff
@@ -72,7 +73,9 @@ const AppContent = () => {
                             onParametersClick={() => console.log("settings")}
                             onLogoutClick={() =>  logout(dispatch, userManager.instance)}
                             onLogoClick={() => console.log("logo")}
-                            user={user} />
+                            user={user}
+                            studyAppColor={appConfig.studyAppColor}
```

Comment:
Why do you pass this to the top bar component, we are not going to add a new property each time we have a new supported app?